.chloggen/receiver_dockerstatsreceiver_add-missing-metrics-for-cgroups-v2.yaml

```diff
@@ -0,0 +1,20 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Just wanted to check why is this marked as breaking?

From what I can tell from the user's perspective these changes are additive?

Reply:
As far as I know, adding a new metric by default is a breaking change.

I added by default this one: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/22706/files#diff-9fde4f9bed227e12ddb4ce32989c6acdb00339e533d8b24688bb43456dc433eaR429

And as I wrote on the PR description:
> I enabled the metric `container.memory.file` by default (causing a breaking change) because it is a metric that was already scraped for cgroups v1, contained in the metric `container.memory.total_cache`.

Let me know if it is not OK and I change it to enhancement.

Reply:
It make sense to me, happy to keep it as breaking.